app/src/editor/shared/components/modules/Map.jsx

```diff
@@ -238,9 +234,23 @@ export default class MapModule extends React.Component<Props, State> {
         const { queuedMarkers } = this
         this.queuedMarkers = {}
 
-        this.setState((state) => ({
-            markers: merge(state.markers, queuedMarkers),
-        }))
+        this.setState((state) => {
+            const markers = { ...state.markers }
+            // $FlowFixMe Object.values() returns mixed[]
```

Comment:
Too bad there's no inline way to get around this, like
`Object.values(queuedMarkers).forEach(((m: any): Marker) => …`

I'd be nice, haha.